tests/FSharp.Test.Utilities/Utilities.fs

```diff
@@ -17,6 +18,19 @@ open NUnit.Framework
 
 module Utilities =
 
+    type Async with
+        static member RunImmediate (computation: Async<'T>, ?cancellationToken ) =
```

Comment:
Shall we consolidate this one with the default one?

Reply:
The default one is not public (not added to FSharp.Core)